redhat/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM registry.access.redhat.com/rhel7
+
+MAINTAINER Cosmo (hello@cloudify.co)
+
+ARG USERNAME
+
+ARG PASSWORD
+
+RUN subscription-manager register --username "$USERNAME" --password "$PASSWORD" --auto-attach
+
+RUN yum -y install python-devel gcc openssl git libxslt-devel libxml2-devel openldap-devel libffi-devel openssl-devel libvirt-devel python-netaddr
+
+RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
```

Comment:
Why not install pip from rpm ?

Reply:
There is no reason for that, but you can check the new commit, I updated it to install from rpm

https://github.com/EarthmanT/cloudify-wagon-builders/pull/2/commits/2c846fba145eba1eeecd6f475b9b986dff00f925